src/index.html

```diff
@@ -8,7 +8,205 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <div id="header" class="header-wrapper">
+        <ul id="main-nav">
+          <li class="main-nav-item"><a href="">Home</a></li>
+        </ul>
+        <div id="nav-side-btn">
+          <button>Download my full resume</button>
+        </div>
+      </div>
+
+      <div id="banner1" class="banner1-wrapper">
+        <div id="banner1-cv-title" class="banner1-cv-title">
+          <div class="banner1-cv-title-name"><span>John Roberts</span></div>
+          <div class="banner1-cv-title-position">
+            <span>BUSINESS EXPERT</span>
+          </div>
+        </div>
+      </div>
+
+      <div id="banner2" class="banner2-wrapper">
+        <div if="banner2-personal-info" class="banner2-personal-info">
+          <div class="avatar-container">
+            <img src="" alt="photo" />
```

Comment:
The `alt` attribute always represents the content of the image, so it should be given a representational name. And for this case, `John Roberts` should be used.